.travis.yml

```diff
@@ -21,3 +22,7 @@
        - gcc-4.8
        - g++-4.8
 
+
+ before_install:
+   - 'case "${TRAVIS_NODE_VERSION}" in 0.*) export NPM_CONFIG_STRICT_SSL=false ;; esac'
```

Comment:
So, this is needed?

Reply:
For the npm that is included in Node.js 0.10.0, yes. That's not the version that is in the PaaS I use, though, but you wanted me to add 0.10.0 to the matrix 👍